ReactWindows/ReactNative.Shared/UIManager/ImportantForAccessibility.cs

```diff
@@ -0,0 +1,32 @@
+namespace ReactNative.UIManager
+{
+    /// <summary>
+    /// Accessibility importance mode. Copied form ReactXP documentation:
+    /// https://microsoft.github.io/reactxp/docs/accessibility.html
```

Comment:
Another ReactXP reference we should get rid of

Reply:
done!